docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -469,4 +469,51 @@ Inf2 instances contain one or more Neuron devices, and each Neuron device includ
 
 </Tip>
 
+
+## Load adapters
+
+### LoRA
+
+Low-Rank Adaptation is fast way to Stable Diffusion to adapt styles of generated images. In Optimum Neuron, we support using one or multiple LoRA adapters by fusing their parameters into the original parameters of the text encoder(s) and the unet during the compilation. Here below is an example of compiling stable diffusion models with LoRA adapters of your choice and using the compiled artifacts to generate styled images:
+
+```python
+
+from diffusers import LCMScheduler
+from optimum.neuron import NeuronStableDiffusionPipeline
+
+
+model_id = "Lykon/dreamshaper-7"
+adapter_id = "latent-consistency/lcm-lora-sdv1-5"
+input_shapes = {"batch_size": 1, "height": 512, "width": 512, "num_images_per_prompt": 1}
+compiler_args = {"auto_cast": "matmul", "auto_cast_type": "bf16"}
+
+# Compile
+pipe = NeuronStableDiffusionPipeline.from_pretrained(
+    model_id,
+    export=True,
+    disable_neuron_cache=True,
```

Comment:
Maybe add a comment to say it is because of a bug that will be fixed in next releases

Reply:
I will put `inline_weights_to_neff=True` (which has the same effect) and a warning there.